myokit/_sim/cvodessim.py

```diff
@@ -361,6 +361,37 @@ def default_state_sensitivities(self):
             return [list(x) for x in self._s_default_state]
         return None
 
+    def state_sensitivities(self):
+        """
+        Returns the sensitivities with respect to state variables, or
+        ``None`` if sensitivities are not enabled.
+        """
+        if self._sensitivities:
+            return [list(x) for x in self._s_state]
+        return None
+
+    def set_state_sensitivities(self, s_values):
+        """
+        Sets the sensitivities with respect to state variables if
+        sensitivities are enabled
+
+        The N x M list of lists 's_values', defines the sensitivities
+        dz/dx for the N state variables and M independent variables
+        """
+        if self._sensitivities:
```

Comment:
Probably wants an error if sensitivities are not enabled (because user seems to think they are: API should be helpful here and complain)